v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/JobSpecMapper.java

```diff
@@ -38,14 +39,25 @@ public class JobSpecMapper {
   private static final Logger LOG = LoggerFactory.getLogger(JobSpecMapper.class);
 
   public static ImportSpecification parse(String jobSpecUri, OptionsParams options) {
-    var json = fetchContent(jobSpecUri);
-    var spec = new JSONObject(json);
+    String content = fetchContent(jobSpecUri);
+    JSONObject spec;
+
+    try {
+      spec = new JSONObject(content);
+    } catch (Exception jsonException) {
```

Comment:
issue(minor): If the user specify malformed json, they won't be able to see the parser error. Instead, they will receive an error about malformed yaml. However, I don't see how this could be avoided without explicit configuration option, which is probably excessive.

Reply:
yes, you're right and it seems only way to avoid is also catch yaml exception and show both errors for json and yaml, I mean something like this; 

```
    try {
      spec = new JSONObject(content);
    } catch (Exception jsonException) {
      Yaml yaml = new Yaml();
      try {
        Map<String, Object> yamlMap = yaml.load(content);
        spec = new JSONObject(yamlMap);

      } catch (Exception yamlException) {
        throw new IllegalArgumentException("Parsing failed: content is neither valid JSON nor valid YAML." +
                "\nJSON parse error: " + jsonException.getMessage() +
                "\nYAML parse error: " + yamlException.getMessage(), yamlException);
      }
    }
```